settings.gradle

```diff
@@ -1,10 +1,13 @@
 rootProject.name = 'kotlinconf-app'
-// include 'web' TODO: Fixme
-//include 'common-js'
+
+enableFeaturePreview('GRADLE_METADATA')
 
 include 'backend'
 include 'common'
-include 'common-jvm'
+include 'konfios'
+
+//include 'web' TODO: Fixme
```

Comment:
if I'm not mistaken, the `common-js` and `web` modules has been removed. Should we also remove the commented out `include`s?

Reply:
Sure